github/event_types.go

```diff
@@ -130,17 +130,6 @@ type GollumEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
-// EditChange represents the changes when an issue, pull request, or comment has
-// been edited.
-type EditChange struct {
-	Title *struct {
-		From *string `json:"from,omitempty"`
-	} `json:"title,omitempty"`
-	Body *struct {
-		From *string `json:"from,omitempty"`
-	} `json:"body,omitempty"`
-}
```

Comment:
@shurcooL , @gmlewis I have made changes for defined structs of ProjectCard, ProjectColumn and project events. Old ones that refer to EditChanges have been kept as old. Please do review and comment.